### PR TITLE
1.5.2 release

### PR DESCRIPTION
**Pull request by tsunamistate**

### CODE REVIEW

File: fix-metainfo-1.5.2.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/Packaging/nix/devilutionx.metainfo.xml b/Packaging/nix/devilutionx.metainfo.xml
+index badf15f5f..5a3bad1f8 100644
+--- a/Packaging/nix/devilutionx.metainfo.xml
++++ b/Packaging/nix/devilutionx.metainfo.xml
+@@ -72,7 +72,7 @@
+ 					<li>Improve rendering of Chinese/Japanese/Korean texts</li>
+ 					<li>Fix Hellfire items not saving their identified state</li>
+ 					<li>Fix stash corrupting when converting a hero between Diablo and Hellfire</li>
+-					<li>Release is now built on Ubuntu 22.04.<br>Warning: This may cause DevilutionX to fail launch on systems with older <b>glibc</b> versions!</li>
++					<li>Release is now built on Ubuntu 22.04.<br/>Warning: This may cause DevilutionX to fail launch on systems with older <b>glibc</b> versions!</li>
```
Comment (wjt) [inline fix-metainfo-1.5.2.patch]: Neither `<br>` nor `<b>` is listed in https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description as supported tags.

If you're patching this line anyway, you could just remove it – it is irrelevant to Flathub – but for upstream the <br> should be removed, and personally I would remove the `<b>` around `glibc` as well.

**Comment by tsunamistate** [comment]
bot, build

**Comment by tsunamistate** [comment]
The patch didn't apply

```
Error: Error on line 75 char 151: Element “li” was closed, but the currently open element is “br”
```

And I'm back to square one

**Comment by tsunamistate** [comment]
bot, build

**Comment by tsunamistate** [comment]
I have given up on patching the archive
Fix was already done upstream https://github.com/diasurgical/devilutionX/pull/6944
After talking with DevX team, there will be another release 1.5 with this fix, and I'll use that

**Pull request closed without merging**